.github/workflows/locker.yml

```
@@ -0,0 +1,36 @@
+name: Locker - Lock stale issues and PRs
+on:
+  schedule:
+    - cron: '0 9 * * *' # Once per day, early morning PT
```

```suggestion
    - cron: '37 8 * * *' # Once per day, early morning PT
```

@terrajobst pointed out that GitHub suggests not using `:00` on cron jobs because it's so commonly used. Picking an arbitrary time.